src/utils/index.js

```diff
@@ -1,10 +1,23 @@
-import { Dimensions } from 'react-native';
+import { Dimensions, NativeModules, Platform } from 'react-native';
 
 export const getFrameDims = (maxWidth, ratio = 9 / 16) => {
   const width = maxWidth || Dimensions.get('window').width;
   const getFrameHeight = w => Math.floor(w * ratio);
-  return ({
+  return {
     width,
-    height: getFrameHeight(width),
-  });
+    height: getFrameHeight(width)
+  };
 };
+
+export const closeModalScreen = Platform.select({
+  android: () => {
+    NativeModules.APReactNativeBridge.handleCommand('stop', {});
+  },
+  ios: () => {
+    NativeModules.ZPReactNativeBridgeListener.postEvent(
+      'dismiss_modal_view',
+      { animated: 1 },
```

Comment:
Maybe it's good idea to add optional param for this animated flag? Even though it is iOS only?

Reply:
Think it should be added in further PR. Might be possible in Android too - need to investigate.